libvast/src/format/simdjson.cpp

```diff
@@ -0,0 +1,449 @@
+  if (real x; parsers::json_number(s, x))
+    return x;
+  return make_error(ec::convert_error, "cannot convert from", std::string{s}, "to real");
+}
+
+// Time conversions
+template <>
+caf::expected<data>
+type_biassed_convert_impl<integer, type_id<time_type>()>(integer s,
+                                                         const type&) {
+  auto secs = std::chrono::duration<real>(s);
+  return time{std::chrono::duration_cast<duration>(secs)};
+}
+
+template <>
+caf::expected<data>
+type_biassed_convert_impl<count, type_id<time_type>()>(count s, const type&) {
+  auto secs = std::chrono::duration<real>(s);
+  return time{std::chrono::duration_cast<duration>(secs)};
+}
+
+template <>
+caf::expected<data>
+type_biassed_convert_impl<real, type_id<time_type>()>(real s, const type&) {
+  auto secs = std::chrono::duration<real>(s);
+  return time{std::chrono::duration_cast<duration>(secs)};
+}
+
+// Duration conversions
+template <>
+caf::expected<data>
+type_biassed_convert_impl<integer, type_id<duration_type>()>(integer s,
+                                                             const type&) {
+  auto secs = std::chrono::duration<real>(s);
+  return std::chrono::duration_cast<duration>(secs);
+}
+
+template <>
+caf::expected<data>
+type_biassed_convert_impl<count, type_id<duration_type>()>(count s,
+                                                           const type&) {
+  auto secs = std::chrono::duration<real>(s);
+  return std::chrono::duration_cast<duration>(secs);
+}
+
+template <>
+caf::expected<data>
+type_biassed_convert_impl<real, type_id<duration_type>()>(real s, const type&) {
+  auto secs = std::chrono::duration<real>(s);
+  return std::chrono::duration_cast<duration>(secs);
+}
+
+// STRING Conversions
+template <>
+caf::expected<data>
+type_biassed_convert_impl<std::string_view, type_id<string_type>()>(
+  std::string_view s, const type&) {
+  return std::string{s};
+}
+
+// ENUMERATION Conversions
+template <>
+caf::expected<data>
+type_biassed_convert_impl<std::string_view, type_id<enumeration_type>()>(
+  std::string_view s, const type& t) {
+  const auto& e = dynamic_cast<const enumeration_type&>(*t);
+
+  const auto i = std::find(e.fields.begin(), e.fields.end(), s);
+  if (i == e.fields.end())
+    return make_error(ec::parse_error, "invalid:", std::string{s});
+  return detail::narrow_cast<enumeration>(std::distance(e.fields.begin(), i));
+}
+
+// ARRAY Conversions
+template <>
+caf::expected<data>
+type_biassed_convert_impl<::simdjson::dom::array, type_id<list_type>()>(
+  const ::simdjson::dom::array& a, const type& t) {
+  const auto& v = dynamic_cast<const list_type&>(*t);
+
+  list xs;
+  xs.reserve(a.size());
+
+  for (const auto& x : a) {
+    if (auto elem = convert(x, v.value_type))
+      xs.push_back(*std::move(elem));
+    else
+      return elem;
+  }
+  return xs;
+}
+
+// MAP Conversions
+template <>
+caf::expected<data>
+type_biassed_convert_impl<::simdjson::dom::object, type_id<map_type>()>(
```

Comment:
```suggestion
type_biased_convert_impl<::simdjson::dom::object, type_id<map_type>()>(
```

Just a nit: It's `biased`, not `biassed`.

Reply:
Fixed.